### PR TITLE
Update how-to-debug-dotnet-framework-source.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**

### CODE REVIEW

**Comment by PRMerger17** [comment]
@Rick-Anderson : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/debugger/how-to-debug-dotnet-framework-source.md [hunk 1]
```diff
@@ -63,3 +63,5 @@ To debug .NET Framework source, you must:
 ### See also
 - [Debugging managed code](../debugger/debugging-managed-code.md)
 - [Specify symbol (.pdb) and source files](../debugger/specify-symbol-dot-pdb-and-source-files-in-the-visual-studio-debugger.md)
+- [Debug .NET and ASP.NET Core source code](/aspnet/core/test/debug-aspnetcore-source)
```
Comment (Rick-Anderson) [inline docs/debugger/how-to-debug-dotnet-framework-source.md:66]:
```suggestion
- [Debug .NET and ASP.NET Core source code with Visual Studio](/aspnet/core/test/debug-aspnetcore-source)
```

**Comment by PRMerger12** [comment]
@Rick-Anderson : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
#sign-off Thanks, Rick!

**Comment by PRMerger8** [comment]
@Rick-Anderson : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson - The link isn't working for me. It looks like it's set up as a site-relative link, but the link targets a destination in a different repo. Could you take another look?
> 

That's how site relative links work for different repo's. Here's the full linke
https://docs.microsoft.com/en-us/aspnet/core/test/debug-aspnetcore-source

How are you testing, I don't see a build report.

**Comment by ShannonLeavitt** [comment]
Thanks for confirming, @Rick-Anderson 

I'll merge now!

**Pull request merged**